src/logic/configure.ts

```diff
@@ -77,41 +74,9 @@ export function handleProviderResponse(
   const security: SecurityValues[] = [];
 
   if (response.securitySchemes) {
-    for (const scheme of response.securitySchemes) {
-      options?.logCb?.(
-        `Configuring ${security.length + 1}/${
-          response.securitySchemes.length
-        } security schemes`
-      );
-      // Char "-" is not allowed in env variables so replace it with "_"
-      const envProviderName = response.name.replace('-', '_').toUpperCase();
-      if (isApiKeySecurityScheme(scheme)) {
-        security.push({
-          id: scheme.id,
-          apikey: `$${envProviderName}_API_KEY`,
-        });
-      } else if (isBasicAuthSecurityScheme(scheme)) {
-        security.push({
-          id: scheme.id,
-          username: `$${envProviderName}_USERNAME`,
-          password: `$${envProviderName}_PASSWORD`,
-        });
-      } else if (isBearerTokenSecurityScheme(scheme)) {
-        security.push({
-          id: scheme.id,
-          token: `$${envProviderName}_TOKEN`,
-        });
-      } else if (isDigestSecurityScheme(scheme)) {
-        security.push({
-          id: scheme.id,
-          digest: `$${envProviderName}_DIGEST`,
-        });
-      } else {
-        options?.warnCb?.(
-          `⚠️  Provider: "${response.name}" contains unknown security scheme`
-        );
-      }
-    }
+    security.push(
```

Comment:
might be nicer do do `security = prepareSecurityValues(...)` directly (with prepareSecurityValues returning [] if response.securitySchemes is not defined) rather than creating an empty array and then pushing into it